client/src/containers/CovidPage/CovidPage.jsx

```diff
@@ -45,14 +49,23 @@ class CovidPage extends Component {
       isFetching: true,
       currentDisplay: 'none',
       locked: false, // lock the sidebar on a state or county
+      steps: COVID_JOYRIDE_STEPS,
+      run: false,
     };
     this.statesRef = React.createRef();
     this.alaskaRef = React.createRef();
     this.hawaiiRef = React.createRef();
     this.mapRef = React.createRef();
   }
 
+  static contextType = MainContext;
+  
   async componentDidMount() {
+
+    const context = this.context;
+    console.log(context);
```

Comment:
upon mounting of the component, change these state values to trigger the joyride to start if the context was changed on the homepage (via the tutorial)